packages/gafl-webapp-service/src/pages/contact/contact/contact.njk

```diff
@@ -94,11 +94,51 @@
     <p class="govuk-body-m">We will show you the licence number on confirmation.</p>
 {% endset -%}
 
+{% set itemsArray = [
+    {
+        value: "email",
+        text: "Email",
+        checked: payload['how-contacted'] === 'email',
+        conditional: {
+            html: emailHtml
+        }
+    },
+    {
+        value: "text",
+        text: "Text message",
+        checked: payload['how-contacted'] === 'text',
+        conditional: {
+            html: textMessageHtml
+        }
+    }
+] %}
+
+{% if data.isPhysical %}
+    {% if data.licensee.postalFulfilment === 'Yes' %}
+        {% set itemsArray = (itemsArray.push(
+            {
+                value: "none",
+                text: "Post",
+                checked: payload['how-contacted'] === 'post'
+            }), itemsArray) 
+        %}
+    {% endif %}
+{% else %}    
+    {% set itemsArray = (itemsArray.push(
+        {
+            value: "none",
+            text: "Make a note of the licence number",
+            checked: payload['how-contacted'] === 'none',
+            conditional: {
+                html: noneMessageHtml
+            }
+        }), itemsArray) 
+    %}
+{% endif %}
+
```

Comment:
Should this sort of logic be in getData? Feels a bit complex for a template.

Reply:
It would be difficult to move this out as it references nunjucks html variables e.g. noneMessageHtml